modules/node_modules/@ncigdc/components/Aggregations/ExactMatchFacet.js

```diff
@@ -82,9 +82,10 @@ const ExactMatchFacet = compose(
                     ], false),
                   }}
                   dark={!!inputValue}
-                  onClick={() => setInputValue('')}
```

Comment:
it was overwritten https://github.com/NCI-GDC/portal-ui/blame/relay/modules/node_modules/%40ncigdc/components/Links/InternalLink.js#L40